### DISCUSSION TITLE
[Config Support]: Snapshot resolution question

### DISCUSSION THREAD

**Discussion by atv2016** [blakeblackshear/frigate#10482] [Config Support]
### Describe the problem you are having

Not really a problem. 
I would like to know why we have the option to crop and give height to snapshots. And if we don't, does it take the resolution of the configured detect: width and height resolution ? if we don't specify those options? At least it looks like it does.

I'm asking because I would like to work with the full resolution snapshot, and it seems like that's very much available (I had those crop and height options set in my snapshot settings). If I configure 4K cameras and set the resolution in detect for that, will I get that resolution in my snapshot image as well ?

Is it just to save diskspace or ?

Final question: how long does the snapshot image stay available. Is the snapshot the file without the -clean.png in the clips directory?

Thanks..

### Version

na

### Frigate config file

```yaml
na
```

### Relevant log output

```shell
NA
```

### Frigate stats

```json
NA
```

### Operating system

Other

### Install method

Docker Compose

### Coral version

USB

**Comment by NickM-27** [comment]
The option is there as some users prefer to lower quality or crop the image so it just shows what was detected. Others detail to see the full scene. If no height is specified (default) then the original detect resolution is used. 

It is highly discouraged to run detect on 4k resolution, this will use a lot of CPU resources and not provide any benefit to object detection accuracy in the majority of cases. Snapshots are kept as long as your snapshot retention period. Snapshots by default are saved as a .jpg with annotations and a .png that is clean.

**Reply by atv2016** [reply]
Cool thanks Nick. Yeah not running detect on 4K, was just wondering if there's any difference between snapshot and the clean if no other options are configured.
